site/content/docs/5.2/migration.md

```diff
@@ -37,22 +37,27 @@ If you need more details about the changes, please refer to the [v5.2.2 release]
 
 - `@mixin button-size()` has a new interface including a new optional parameter for button letter spacing.
 
+- <span class="badge bg-danger">Breaking</span> `$form-range-thumb-active-border` doesn't exist any more. It has been replaced by `$form-range-thumb-active-border-color`.
```

Comment:
Maybe we can present it as a list like the new CSS/Sass variables.

Reply:
You mean in a <details> tag, with a title such as "Deprecated Sass variables" ? 
Is it to have a presentation alike the one used for the new CSS/Sass variables ? In this case, shall I delete the danger icon ?